common/plugins/org.jboss.tools.common.core/src/org/jboss/tools/common/core/validation/InetAddressValidator.java

```diff
@@ -0,0 +1,184 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
So you didn't modify the original code (besides changing package name)? I'm not sure this is the best solution. If we copy sources just to use them as is then we will have to maintain this code etc.